redis/asyncio/cluster.py

```diff
@@ -319,7 +339,6 @@ def __init__(
         self.reinitialize_steps = reinitialize_steps
         self.cluster_error_retry_attempts = cluster_error_retry_attempts
         self.connection_error_retry_attempts = connection_error_retry_attempts
-
```

Comment:
Nitpick: Don't delete this line.

Reply:
I didn't delete it, just moved it to line 309